collection-scripts/gather_main

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bash
+
+export BASE_COLLECTION_PATH="must-gather"
+mkdir -p ${BASE_COLLECTION_PATH}
+
+# Command line argument
+export SINCE_TIME=$1
+
+# Source the utils
+# shellcheck disable=SC1091
+. utils.sh
+
+# timestamp for starting of the script
+START_TIME=$(date +%r)
+start=$(date +%s)
+dbglog "collection started at: ${START_TIME}"
+
```

Comment:
Looks like this is same file as main:collection-scripts/gather. If that is the case, would you please use git mv to rename the file instead of adding it as a new file. Adding it as a new file, looses all the history of SCM.